docs/advanced/manual-cli-install.md

```diff
@@ -1,77 +1,96 @@
-
-1. If the operating system dependent home directories listed in the [configuration files](#configuration-file-locations) section above do not exist, please create them for `dcrd`, `dcrwallet`, and `dcrctl`.
-2. Using your favourite text editor, create a new file enter the text, `[Application Options]` as the first line. Save it as `dcrd.conf` in `dcrd`'s home directory.
-3. Create another new file and add the text `[Application Options]` to the first line as before. Save it as `dcrwallet.conf` in `dcrwallet`'s home directory.
-4. Create a third text file and save it as `dcrctl.conf` in `dcrctl`'s home directory. You don't need to add anything to this one right now.
-5. Choose an arbitrary username and password, these will only be used for each application to communicate via remote procedure call. The easiest configuration is to set them all equal.
-6. Inside `dcrd.conf`, underneath `[Application Options]`, add the following lines:<br /><br />
-        `rpcuser=<chosen-username>`<br />
-        `rpcpass=<chosen-password>`<br /><br />
-7. Inside `dcrwallet.conf`, underneath `[Application Options]`, add the following lines:<br /><br />
-        `username=<chosen-username>`<br />
-        `password=<chosen-password>`<br /><br />
-8. Inside `dcrctl.conf`, add the following lines:<br /><br />
-        `rpcuser=<chosen-username>`<br />
-        `rpcpass=<chosen-password>`<br /><br />
-9. Save all three configuration files.
-
-Please see the [sample config](https://github.com/decred/dcrd/blob/master/sampleconfig/sampleconfig.go#L8-L352) for more customisation options.
\ No newline at end of file
+At the very minimum, for `dcrd`, `dcrwallet`, and `dcrctl` to be able to communicate with each other, they need to be launched with the same `rpcuser`/`rpcpass` combination. For manual configuration please follow these steps:
+
+1. If the OS-specific home directories listed in [Configuration File Locations](/getting-started/startup-basics#configuration-file-locations) do not exist, please create them for `dcrd`, `dcrwallet`, and `dcrctl`.
+
+2. Choose an arbitrary username and password, these will only be used for each application to communicate via remote procedure call. The easiest configuration is to set them all equal.
+
+3. Using your favorite text editor, create a new text file and add the following lines:
+
+        [Application Options]
+        rpcuser=<chosen-username>
+        rpcpass=<chosen-password>
+
+    Save it as `dcrd.conf` in `dcrd`'s home directory.
+
+4. Create another new text file and add the following lines:
+
+        [Application Options]
+        username=<chosen-username>
+        password=<chosen-password>
+
+    Save it as `dcrwallet.conf` in `dcrwallet`'s home directory.
+
+5. Create a third text file and add the following lines:
```

Comment:
Does this need [Application Options] as well?

Reply:
They aren't required but I'll add it for consistency.